src/frontend/src/views/Faq/index.tsx

```diff
@@ -181,31 +167,20 @@ export default function Faq(): JSX.Element {
           </span>
         </ListItem>
         <ListItem>
-          Only other members of your group can see your data. CDPH can see
-          samples, but not your private, internal identifiers.
-        </ListItem>
-        <ListItem>
-          New sequences will be automatically submitted to GISAID two weeks
-          after upload, unless marked &quot;private.&quot;
-        </ListItem>
-        <ListItem>
-          <span>
-            You can mark a sample as &quot;private&quot; anytime during the
-            first two weeks after upload. &quot;Private&quot; samples are not
-            shared with CDPH or GISAID, but are visible to your group. (For now,
-            please send us a quick{" "}
-            <NewTabLink href="mailto:helloaspen@chanzuckerberg.com">
-              email
-            </NewTabLink>{" "}
-            to mark samples as &quot;private.&quot;)
-          </span>
+          Only other members of your group can see your data. Other
+          organizations that you share your data with can see your samples, but
+          not your private, internal identifiers.
         </ListItem>
         <ListItem>
-          CZ GEN EPI does not support protected health information.
+          You can mark a sample as “private” anytime. &quot;Private&quot;
```

Comment:
why are there two different kinds of quotes?

Reply:
so for some reason the linter was only complaining about _some_ of the quote characters?

Reply:
ooh, I wonder if it was complaining about straight (") quotes only, and not curly (“”).